bee-ledger/src/workers/consensus/white_flag.rs

```diff
@@ -177,8 +185,10 @@ async fn traverse_past_cone<B: StorageBackend>(
             if let Some(unvisited) = message.parents().iter().find(|p| !visited.contains(p)) {
                 message_ids.push(*unvisited);
             } else {
-                apply_message(storage, message_id, &message, metadata)?;
-                visited.insert(*message_id);
+                if !visited.contains(message_id) {
```

Comment:
:thinking: bug-fix? :trollface:

Reply:
Yes an no, this is now required because the milestone is not confirmed by itself